tests/common.go

```diff
@@ -4042,6 +4043,25 @@ func GetAzureCredsFromEnv() (tenantID, clientID, clientSecret, subscriptionID, a
 	return tenantID, clientID, clientSecret, subscriptionID, accountName, accountKey
 }
 
+// GetNfsCredsFromEnv get creds for nfs share.
+func GetNfsCredsFromEnv() (nfsServerAddress, nfsPath, nfsSubPath, nfsMountOptions string) {
```

Comment:
It is not good to return 3 different values from a function. Can you create a struct called `NfsInfo` and make `nfsServerAddress, nfsPath, nfsSubPath, nfsMountOptions` the attributes

Reply:
Done.